runtime/include/json.h

```diff
@@ -254,6 +258,19 @@ parse_json(const char *json_buf, ssize_t json_buf_size, struct module_config **m
 				int rc = parse_uint32_t(tokens[i], json_buf, module_keys[module_expected_execution_us],
 				                        &(*module_config_vec)[module_idx].expected_execution_us);
 				if (rc < 0) goto json_parse_err;
+			} else if (strcmp(key, module_keys[module_replenishment_period_us]) == 0) {
```

Comment:
Unfortunately, I'm sure this will all conflict with @bushidocodes updates to the json.

Reply:
Yes.